simple/sample/output/debug/triples.csv

```diff
@@ -1,4 +1,304 @@
 subject_id,predicate,object_id,object_value
+JG497095,typeOf,CrimeEvent,""
```

Comment:
do we need to store the entire table as debug csv? That feels too much

Reply:
We actually don't need to store the `triples.csv` at all anymore. I'll updated the code so it's no longer generated.

This one however, is just the [shell script](https://github.com/datacommonsorg/import/blob/master/run_test.sh#L108) saving the tables as triples. This is to ensure that the expected changes are correctly captured and that the tables don't change when they are not expected to.

I think the name `debug` is a misnomer here. So have changed it to `tables` so it's more indicative.

Reply:
I see